vdslib/src/main/java/com/yahoo/vdslib/distribution/Distribution.java

```diff
@@ -122,7 +127,11 @@ public Distribution(StorDistributionConfig config) {
     }
 
     public void close() {
-        if (configSub!=null) configSub.close();
+        if (configSub!=null) {
+            configSub.close();
+            configSub = null;
```

Comment:
Setting references to null is generally not recommended, but I assume this is one of the cases where it really has an impact. I suggest to leave a comment so that other developers don't start setting references to null unnecessarily.
(https://stackoverflow.com/questions/449409/does-assigning-objects-to-null-in-java-impact-garbage-collection)